ansible/mnesia_backup.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Clarify whether this part.

Reply:
The file was added by accident. Now removed.